pkgs/applications/editors/emacs/elisp-packages/manual-packages/elpaca/default.nix

```
@@ -0,0 +1,24 @@
+{
+  melpaBuild,
+  fetchFromGitHub,
+  git,
+}:
+
+melpaBuild {
+  pname = "elpaca";
+  version = "0.0.1";
+
+  src = fetchFromGitHub {
+    owner = "progfolio";
+    repo = "elpaca";
+    rev = "db2fd7258ff69fe2d100888cb8d92cf3bf94d465";
+    hash = "sha256-SseY0iU3D3cloKZy6xPp8QT0H1Cu2uGiiVG6rXq/UHg=";
+  };
+
+  packageRequires = [ git ];
+
```

An update script makes our bot update this package for you.
```suggestion
  passthru.updateScript = unstableGitUpdater { };
```